server/odc-service/src/main/java/com/oceanbase/odc/service/task/executor/TaskApplication.java

```diff
@@ -24,6 +24,7 @@
 
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.core.LoggerContext;
+import org.eclipse.jgit.util.StringUtils;
```

Comment:
is apache.common.StringUtils you wanted?

Reply:
Yes I should use `com.oceanbase.odc.common.util.StringUtils`. Fixed it now.